### PR TITLE
Add Brazilian Portuguese localizations

### PR DESCRIPTION
**Pull request by glasswalk3r**
No Jira related task.

### Proposed changelog entries

* Add Brazilian Portuguese localizations.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [X] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

None that I'm aware of.

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by basil — APPROVED** [review]

File: core/src/main/resources/jenkins/security/ApiTokenProperty/config_pt_BR.properties [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+CurrentTokens=Passe(s) atual(is)
+LegacyToken=Recomendamos fortemente que voc\u00EA revogue este passe legado e o troque por um novo rec\u00E9m gerado para aumentar a seguran\u00E7a.
+ConfirmRevokeSingle=Voc\u00EA tem certeza de que quer revogar este passe? Aplica\u00E7\u00F5es que o estejam usando n\u00E3o v\u00E3o conseguir se conectar mais.
+Cancel=Cancelar
+StatisticsDisabled.Title=Estat\u00EDsticas de use de passe est\u00E3o atualmente desabilitadas
+TokenCreation=Criado {0} dia(s) atr\u00E1s
+Legacy\ API\ Token=Passe\ de\ API\ legado
+TokenNeverUsed.Title=Recomendamos fortemente que voc\u00EA revogue os passes que voc\u00EA n\u00E3o planeja utilizar
+StatisticsDisabled=Nenhuma estat\u00EDstica dispon\u00EDvel
+TokenNeverUsedTooltip=---TranslateMe 698--- There is no last use date for that token
```
Comment (timja) [inline core/src/main/resources/jenkins/security/ApiTokenProperty/config_pt_BR.properties:35]: is this intentional?

File: core/src/main/resources/jenkins/security/apitoken/Messages_pt_BR.properties [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+LegacyApiTokenAdministrativeMonitor.displayName=Uso de passe de API legado
+ApiTokenPropertyDisabledDefaultAdministrativeMonitor.displayName=Passe de API legado n\u00E3o \u00E9 gerado por padr\u00E3o
+---TranslateMe 630--- Legacy API Token not generated by default
```
Comment (timja) [inline core/src/main/resources/jenkins/security/apitoken/Messages_pt_BR.properties:25]: is this right?

File: core/src/main/resources/jenkins/security/ApiTokenProperty/config_pt_BR.properties (hunk 1 above)
Comment (glasswalk3r) [reply core/src/main/resources/jenkins/security/ApiTokenProperty/config_pt_BR.properties]: Absolutely not, I just fixed and push again.

File: core/src/main/resources/jenkins/security/apitoken/Messages_pt_BR.properties (hunk 2 above)
Comment (glasswalk3r) [reply core/src/main/resources/jenkins/security/apitoken/Messages_pt_BR.properties]: Absolutely not, I just fixed and push again.

**Review by timja — APPROVED** [review]
not checked the translations

**Pull request merged**